mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/pdl/PdlOpplysning.kt

```diff
@@ -53,3 +53,30 @@ enum class PdlGradering {
     STRENGT_FORTROLIG_UTLAND,
     UGRADERT,
 }
+
+sealed class GeografiskTilknytningResponse {
```

Comment:
Fungerer det om vi bare kaller denne for `GeografiskTilknytning`( evt. `GeografiskTilknytningPerson` hvis dette bare gjelder for personer), eller er det ønskelig å ha med at dette er `Response`? 

Selv tolker jeg typen `PdlGeografiskTilknytning` som den egentlige responsen (fra pdl), så da synes det virker litt rart å mappe den typen om til et nytt response-objekt (med forbehold om at jeg kanskje ikke ser helheten her).